### ISSUE TITLE
Add Laravel 10 Support

### ISSUE THREAD

**Issue by drice82** [matriphe/laravel-md5-hash#25]
The laravel-md5-hash helps me a lot, but it cannot upgrade to Laravel 10 now.
Can you add Laravel 10 Support? Thanks.

**Comment by oracleh2** [comment]
Agree, I tested it in Laravel 10 with no problem, please fix your composer.json just adding laravel 10 support.
